src/primitives/block.h

```diff
@@ -64,12 +64,13 @@ class DynaFedParamEntry
     CScript m_fedpeg_program; // The "scriptPubKey" of the fedpegscript
     CScript m_fedpegscript; // The witnessScript for witness v0 or undefined otherwise.
     // No consensus meaning to the particular bytes, currently we interpret as PAK keys, details in pak.h
-    std::vector<std::vector<unsigned char>> m_extension_space;
+    std::vector<std::vector<unsigned char>> m_extension_space;\
+    uint256 m_elided_root; // non-zero only when m_serialize_type == 1
```

Comment:
Perhaps document the new field to explain how the compact vs full params work? Also you have a type `\` there.